.gitignore

```diff
@@ -1,3 +1 @@
-build
-build-test
-dist
+
```

Comment:
Why are you removing these from Git's ignore list?
They are temporary directories which contain the compiler output. So as they're temporary local stuff they shouldn't be committed to the repository. Knowing that, do you still have any reason to remove them?